pkg/tsdb/cloudwatch/response_parser_test.go

```diff
@@ -272,11 +274,13 @@ func Test_buildDataFrames_parse_label_to_name_and_labels(t *testing.T) {
 		require.NoError(t, err)
 
 		assert.Equal(t, "some label lb3", frames[0].Name)
+		assert.Equal(t, "some label lb3", frames[0].Fields[1].Labels["Series"])
 		assert.Equal(t, "balancer 1", frames[0].Fields[1].Labels["LoadBalancer"])
 		assert.Equal(t, "inst1", frames[0].Fields[1].Labels["InstanceType"])
 		assert.Equal(t, "tg", frames[0].Fields[1].Labels["TargetGroup"])
 
 		assert.Equal(t, "some label lb4", frames[1].Name)
+		assert.Equal(t, "some label lb4", frames[1].Fields[1].Labels["Series"])
```

Comment:
I decided not to add this for all the parser tests because it felt redundant and it wasn't the thing those tests were checking, but I can add them if we'd prefer